_posts/2016-05-20-drupalcon-nola-recap.md

```diff
@@ -0,0 +1,118 @@
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Putting my 💰 where my 🤑 is and investing in <a href="https://twitter.com/hashtag/Drupal?src=hash">#Drupal</a>:  <a href="https://t.co/s1aUuoKxkv">https://t.co/s1aUuoKxkv</a> <a href="https://twitter.com/DrupalConNA">@DrupalConNA</a> code sprinting 🏃 ! <a href="https://t.co/mlSXsZif4m">pic.twitter.com/mlSXsZif4m</a></p>&mdash; Savas Labs (@Savas_Labs) <a href="https://twitter.com/Savas_Labs/status/731148211218436097">May 13, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+It is in the track world just as it is in the coding world: **sprinting is hard!**. It is difficult to get organized, get oriented (or help orient) and contribute effectively to a global project in a narrow window of time.
+
+Given my newfound desire to ease the barriers and access to contributing to the project in general, I decided to sprint to make the drupal.org contributor experience better. In doing so, I got [my very own installation of the drupal.org website](https://chrisarusso-drupal.dev.devdrupal.org/) of which I had complete control. **Fun**!
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Just for one glorious day when contributing to <a href="https://twitter.com/hashtag/Drupal?src=hash">#Drupal</a>, I AM <a href="https://twitter.com/Dries">@Dries</a>! <a href="https://twitter.com/DrupalConNA">@DrupalConNA</a> <a href="https://twitter.com/hashtag/DreamsRealized?src=hash">#DreamsRealized</a> <a href="https://t.co/aWGlGaM67P">pic.twitter.com/aWGlGaM67P</a></p>&mdash; Savas Labs (@Savas_Labs) <a href="https://twitter.com/Savas_Labs/status/731184271218610177">May 13, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+<span class="caption">Note: Dries liked this Tweet</span>
+
+As mentioned, **sprinting is hard**, so I was actually unable to meaningfully contribute a patch during my stay before I had to head to the airport.
+
+However, I was able to follow up shortly after returning home to [submit a patch](https://www.drupal.org/node/2035235#comment-11201809) to the issue I was working on which is currently in review. Exciting stuff.
+
+### Wrap-up
+
+In conclusion, one of my biggest takeaways from the whole experience is that **it's all about the community!!** ... that's what makes Drupal great. Some of my favorite people on planet earth I either work with, or met through Drupal. Conferences are a nice time to connect in the flesh with people who we know on the other side of a video call, or less yet, as a username on a website. New Orleans proved to be a good city to host Drupalcon at, and we're already looking forward to Baltimore next year.
```

Comment:
I think "Earth" should be capitalized. Per Google `When you're talking about the planet we live on, capitalize the word.` I would also remove the "at" at the end of `New Orleans proved to be a good city to host Drupalcon at`, but that's just personal preference.